cosmosdb/sql/ps-container-create-index-none.ps1

```diff
@@ -60,4 +60,4 @@ $ContainerProperties = @{
 
 New-AzResource -ResourceType $containerResourceType `
```

Comment:
This command is now failing:

```
New-AzResource : {"code":"NotFound","message":"Message: {\"code\":\"NotFound\",\"message\":\"Message: {\\\"Errors\\\":[\\\"Resource Not Found\\\"]}\\r\\nActivityId: 0b9cf458-ea24-46b6-8022-8754b7c925aa
, Request URI: /apps/24e0c5ba-5ce4-45a1-98ff-27922cebbeee/services/52eac7ed-2f4b-40af-8b72-6f5995b75421/partitions/f9534c83-87fe-4b7a-a3e0-0a6e77d4b05d/replicas/132132990133149960s, RequestStats: \\r\\
nRequestStartTime: 2019-09-18T17:33:18.0154163Z, RequestEndTime: 2019-09-18T17:33:18.0154163Z, Number of regions attempted: 1\\r\\nResponseTime: 2019-09-18T17:33:18.0154163Z, StoreResult: StorePhysicalAddress: rntbd://10.0.0.27:17000/apps/24e0c5ba-5ce4-45a1-98ff-27922cebbeee/services/52eac7ed-2f4b-40af-8b72-6f5995b75421/partitions/f9534c83-87fe-4b7a-a3e0-0a6e77d4b05d/replicas/132132990133149960s, LSN: 8, GlobalCommittedLsn: 8, PartitionKeyRangeId: , IsValid: True, StatusCode: 404, SubStatusCode: 0, RequestCharge: 1, ItemLSN: -1, SessionToken: -1#8, UsingLocalLSN: False, TransportException: null, ResourceType: Collection, OperationType: Read\\r\\nResponseTime: 2019-09-18T17:33:18.0154163Z, StoreResult: StorePhysicalAddress: rntbd://10.0.0.33:19700/apps/24e0c5ba-5ce4-45a1-98ff-27922cebbeee/services/52eac7ed-2f4b-40af-8b72-6f5995b75421/partitions/f9534c83-87fe-4b7a-a3e0-0a6e77d4b05d/replicas/132132990133149961s, LSN: 8, GlobalCommittedLsn: 8, PartitionKeyRangeId: , IsValid: True, StatusCode: 404, SubStatusCode: 0, RequestCharge: 1, ItemLSN: -1, SessionToken: -1#8, UsingLocalLSN: False, TransportException: null, ResourceType: Collection, OperationType: Read\\r\\n, SDK: Microsoft.Azure.Documents.Common/2.5.1\"}, Request URI: /dbs/database1/colls/container1, RequestStats: , SDK: Microsoft.Azure.Documents.Common/2.5.1"}
At /home/stramer/src/azure-docs-powershell-samples/cosmosdb/sql/ps-container-create-index-none.ps1:61 char:1
+ New-AzResource -ResourceType $containerResourceType `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : CloseError: (:) [New-AzResource], ErrorResponseMessageException
+ FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.NewAzureResourceCmdlet
```

Unsure of what's causing the issue here offhand, looks like passing the wrong name for a resource somewhere?

Reply:
fixed.